game/outfits.rpy

```diff
@@ -1113,7 +1113,7 @@ init -1 python in jn_outfits:
     ))
     __register_wearable(JNHairstyle(
         reference_name="jn_hair_twintails_long",
-        display_name="Twintails long",
+        display_name="Long Hair with twintails",
```

Comment:
Almost! Sorry to be a pain, but do you think you could lowercase the "h" in "Hair"? Just so it's all consistent when we see it in a list with the others:

![image](https://user-images.githubusercontent.com/57731669/192153987-8afd1975-5b14-4ba5-aa4c-e3292e52a4ba.png)

Reply:
Damn you German genes! Sure, will not take long